### DISCUSSION TITLE
God bless you

### DISCUSSION THREAD

**Discussion by 4lt3r3go** [ilian6806/stable-diffusion-webui-eyemask#10] [Show and tell]
Dude, whoever you are wish you best from life.
I forgot how many times i complained about no one ever thought about making this extention, wich is a hell of essential.
Basics of photography and realism, were one focus on eyes first of all, and all time wasted for inpainting... omg..
thank you. You are an hero

**Comment by 4lt3r3go** [comment]
just a question, not sure if i really need to fill the prompt box, does it take the prompt from above and use that if i do not write anything?

**Reply by ilian6806** [reply]
If you left the mask prompt empty it uses original prompt for redrawing :)

**Reply by 4lt3r3go** [reply]
> If you left the mask prompt empty it uses original prompt for redrawing :)

uhm i'm not that sure, i mean:
-script denoise at 0,75 as test
-empty prompt gives a totally smudged face
-copying/pasting the same prompt from above give the right result

are you sure is like you say?

**Reply by ilian6806** [reply]
Sorry, my mistake, you are right. If you leave it empty, it will be empty. I am not sure if it is a good idea to always reuse the original prompt. I can add it as an option. Also, you can use the "Save last mask prompt' setting if you use similar mask prompts.
